Tasks/MavenV3/task.loc.json

```diff
@@ -412,6 +412,48 @@
       "defaultValue": "false",
       "groupName": "CodeAnalysis",
       "helpMarkDown": "ms-resource:loc.input.help.findbugsAnalysisEnabled"
+    },
+    {
+      "name": "spotbugsAnalysisEnabled",
+      "aliases": [
+        "spotbugsAnalysis"
+      ],
+      "type": "boolean",
+      "label": "ms-resource:loc.input.label.spotbugsAnalysisEnabled",
+      "required": false,
+      "defaultValue": "false",
+      "groupName": "CodeAnalysis",
+      "helpMarkDown": "ms-resource:loc.input.help.spotbugsAnalysisEnabled"
+    },
+    {
+      "name": "spotbugsMavenPluginVersion",
+      "aliases": [
+        "spotbugsMavenPluginVersion"
+      ],
+      "type": "string",
+      "label": "ms-resource:loc.input.label.spotbugsMavenPluginVersion",
+      "required": false,
+      "defaultValue": "4.5.3.0",
+      "helpMarkDown": "ms-resource:loc.input.help.spotbugsMavenPluginVersion",
+      "groupName": "CodeAnalysis",
+      "visibleRule": "spotBugsAnalysisEnabled = true"
+    },
+    {
+      "name": "spotbugsGoal",
+      "aliases": [
+        "spotbugsGoal"
```

Comment:
Does it make sense to specify alias if it's the same as name?

Reply:
Fixed, thanks!